.travis.yml

```diff
@@ -14,4 +14,4 @@ before_install:
   - go get github.com/modocache/gover
 
 script:
-  - bash test.sh
+  - sudo PATH=$PATH GOPATH=$GOPATH GOROOT=$GOROOT bash test.sh
```

Comment:
Rather than requiring sudo to run the tests, let's change the VA test to run in TestMode, so it can validate on a non-443 port.